### PR TITLE
Fix #993 issue

### LINKED ISSUE #993: 5.2 'Tips to Improve Data Visualization' content needs to improve
https://github.com/mschermann/data_viz_reader/blob/807bfeb092a95668c952ba8769deaf278ef7d5a5/04-patterns.Rmd#L44

Reader mentioned 7 important elements on improving data visualization, I also think graph integrity is a very important element should be add on, because an overblown or understated message will mislead audience. So graph integrity is very important.

### PR DESCRIPTION
**Pull request by Cindyli0310**
I add some detailed information about graph integrity explanation, examples, reference book  to better address this criteria.

### CODE REVIEW

**Comment by mschermann** [comment]
Please ask a classmate for the review. I am only reviewing PR into master! Thanks!

File: 04-patterns.Rmd [hunk 1]
```diff
@@ -73,6 +73,11 @@ While style is one form of customization, layering unique data sets on a single
 ### Keep It Simple
 Analytic results shouldn’t be presented to 10 decimal places when the user doesn’t need that level of precision to make a decision or understand a concept. Effective visual interfaces avoid 3-D effects or ornate gauge designs (a.k.a. “chart junk”) when simple numbers, maps or graphs will suffice.
 
+### Graph Integrity
+"Graphics were used as instruments for reasoning quantitative information. With this good exmaple ,graphical workd has come to flourish.Graphical excellence begin with telling the truth about the data(Tufte,2001)".
```
Comment (preranap1) [inline 04-patterns.Rmd]:
```suggestion
"Graphics were used as instruments for reasoning quantitative information. With this example, graphical work has come to flourish. Graphical excellence begins with telling the truth about the data (Tufte,2001)".
```

File: 04-patterns.Rmd [hunk 2]
```diff
@@ -73,6 +73,11 @@ While style is one form of customization, layering unique data sets on a single
...
+In Tufte's "The Visual Display of Quantitative Information" book ,he explained how to create a graph integrity visualization with a whole chapter, vey useful and important examples can be found in the book. Following are some examples:
```
Comment (preranap1) [inline 04-patterns.Rmd]:
```suggestion
In Tufte's "The Visual Display of Quantitative Information" book, an entire chapter is dedicated to graphical integrity in visualizations. Very useful and important examples can be found in the book. Following are some examples:
```

**Comment by btamsir** [comment]
Hi Prerana, 

I also think it is much clearer so great job!

**Comment by Cindyli0310** [comment]
> Hi Prerana,
> 
> I also think it is much clearer so great job!

Hi @btamsir , do you approve the review if you think the changes are good? Because it is still showing awaiting for response on my side.

**Pull request merged**